docs/source/roadmap.rst

```diff
@@ -64,6 +59,14 @@ Submit a PR!
 Release History
 ---------------
 
+.. rubric:: Done in v3.3
+
+* **RHEL/CentOS:** Drop support for RHEL/CentOS 6.x.
+* **Mistral deprecation:** Orquesta replaces Mistral as the workflow engine.
+* **HipChat Removal:** The HipChat adapter has been removed from st2chatops.
+* **Chef:** Deprecated `chef-stackstorm <https://github.com/stackstorm/chef-stackstorm>`_ deployment.
```

Comment:
We didn't mention the chef or docker in the release blog, should we given they are listed here as changes?

Reply:
Added to the blog post